### DISCUSSION TITLE
how to get rid of "ysh is not yet supported" start prompt

### DISCUSSION THREAD

**Discussion by stofey** [starship/starship#6370] [Q&A] [answered]
im on MacOS 15.0.1 using zsh and for a week now every time I open Alacritty I get this prompt and I don't really know how to get rid of it, I don't have a starship config I just use the standard config. Thanks in advance
<img width="676" alt="Screenshot 2024-11-04 at 11 09 45" src="https://github.com/user-attachments/assets/d41f68bf-1d73-45b0-8293-93fa652cab7e">

**Comment by davidkna** [comment, marked as answer]
Please verify that you are running `starship init zsh` and not `starship init ysh` in your zshrc. If the issues persists please post your zshrc.

**Reply by stofey** [reply]
that was it thanks, for some reason I had starship init zsh and ysh in my zshrc. I guess that's what I get for blindly copying stuff from YouTube while knowing nothing about it. its fixed now thanks
